### ISSUE TITLE
Support scripted

### ISSUE THREAD

**Issue by dwijnand** [coursier/sbt-launcher#21]
I haven't tested a minimisation of the problem but I think `scripted` doesn't work with this.

I'm seeing

```
[error] Error during sbt execution: Could not finder sbt launch configuration.  Searched classpath for:
[error] 	/sbt.boot.propertiesUnspecified
[error] 	/sbt/sbt.boot.propertiesUnspecified
[error] 	/sbt.boot.properties
[error] 	/sbt/sbt.boot.properties
```

**Comment by dwijnand** [comment]
See https://github.com/coursier/coursier/issues/650

**Comment by alexarchambault** [comment]
Did that happen in a particular project? `sbt "sbt-coursier/scripted shared-2/simple"` works from the sources of [sbt-coursier](https://github.com/coursier/sbt-coursier).

Another thing I'm not sure is whether coursier fetches the sbt version scripted uses, and whether sbt-coursier is loaded in the sbt sessions that scripted starts.

**Comment by dwijnand** [comment]
Yeah happened in https://github.com/playframework/interplay, on the current master HEAD https://github.com/playframework/interplay/commit/d782b752a22657221648770f1d2512eeda566eec

**Comment by alexarchambault** [comment]
Turned out this was an issue in sbt-coursier, along the lines of https://github.com/coursier/coursier/issues/650 (that you linked above). The sbt launcher dependency was duplicated in `allDependencies`, which was making the workaround of https://github.com/coursier/coursier/issues/650 not work anymore.

https://github.com/coursier/sbt-coursier/pull/42 has a fix for it (first commit).

**Comment by alexarchambault** [comment]
https://github.com/coursier/sbt-launcher/pull/22 should also add a better scripted support here, when one is using sbt >= 1.2.0 at least. It's included in `1.2.3` (released earlier today).

**Comment by dwijnand** [comment]
Thanks!

**Comment by dwijnand** [comment]
TODO: rerun scripted in the interplay with latest coursier/sbt-launcher and verify it doesn't fall over any more.
